packages/vue/src/composables/useAuth.ts

```diff
@@ -53,20 +52,25 @@ export const useAuth = createSharedComposable((): UseAuth => {
 export const useAuthenticator = createSharedComposable(() => {
   const { authStatus, state, send } = useAuth();
 
-  const useAuthenticatorValue = reactive({
-    ...facade,
-    send,
-    state,
-  }) as any;
```

Comment:
This initial value is not needed, because `watchEffect` is guaranteed to run immediately (see comment below). And `...facade` was spreading a mock value anyway, so we aren't losing anything.